.env.example

```
@@ -1,3 +1,12 @@
+# Email address for the personal information section
 NUXT_PUBLIC_EMAIL=
+# Phone number for the personal information section
 NUXT_PUBLIC_PHONE_NUMBER=
-NUXT_PUBLIC_ADDRESS=
\ No newline at end of file
+# Website name for SEO purposes
+NUXT_SITE_NAME=
+# Website URL for SEO purposes
+NUXT_SITE_URL=
+# Whether the environment is development, staging or production
+NUXT_SITE_ENV=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NUXT_SITE_ENV key should go before the NUXT_SITE_NAME key

(UnorderedKey)